rerun_py/rerun_sdk/rerun/archetypes/tensor_ext.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any, Sequence
+
+if TYPE_CHECKING:
+    from ..datatypes import TensorDataLike
+    from ..datatypes.tensor_data_ext import TensorLike
+
+
+class TensorExt:
+    def __init__(
+        self: Any,
+        data: TensorDataLike | TensorLike | None = None,
+        *,
+        names: Sequence[str | None] | None = None,
+    ):
+        """
+        Construct a `Tensor` archetype.
+
+        The `Tensor` archetype internally contains a single component: `TensorData`.
+
+        See the `TensorData` constructor for more advanced options to interpret buffers
+        as `TensorData` of varying shapes.
+
+        For simple cases, you can pass array objects and optionally specify the names of
+        the dimensions. The shape of the `TensorData` will be inferred from the array.
+
+        Parameters
+        ----------
+        self:
+            The TensorData object to construct.
+        data: TensorDataLike | None
+            A TensorData object, or type that can be converted to a numpy array.
+        names: Sequence[str] | None
```

Comment:
maybe `dim_names`  would be a slightly clearer name? I don't want it to be too verbose, but ideally parameters should be understandable from just their name, and `names` doesn't really clue you in _what_ they are naming